kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -138,16 +138,9 @@ export class BrowserInterface {
     if (newWindow != null) newWindow.opener = null
   }
 
-  public PerformanceHiccupReport(data: { hiccupsInThousandFrames: number; hiccupsTime: number; totalTime: number }) {
-    queueTrackingEvent('hiccup report', data)
-  }
-
-  public PerformanceReport(data: { samples: string; fpsIsCapped: boolean }) {
+  public PerformanceReport(data: { samples: string; fpsIsCapped: boolean, hiccupsInThousandFrames: number; hiccupsTime: number; totalTime: number }) {
     const perfReport = getPerformanceInfo(data)
```

Comment:
```suggestion
    const perfReport = getRawPerformanceInfo(data)
```

If we are going to keep one of the events, please let's keep the raw getter and remove the `getPerformanceInfo` method altogether. Kernel shouldn't be calculating percentiles, that should be done from the query on whatever service we use to consume the data (i.e. metabase or grafana).

Reply:
@BrianAmadori are you sure? The raw event is certainly less CPU usage, but it uses much more network bandwith instead (Sends an array with 1000 numbers), i'm fine with either.

Reply:
The data ingestion in that case would be at least 1000x more expensive. The only thing we need from histograms are percentiles, total samples and s of all samples. Everything else can be calculated in the reporting tools. Raw data is not convenient or more useful than those bits of information.

Reply:
I've been testing this function locally and it takes about 0.05ms to run in my (admitedly fast intel i9) notebook, and this would run only once every 1000 samples, do we really need to optimize this on the CPU side instead of bandwitdth? Thanks!

Reply:
The network usage points feel valid to me, those are strong concerns. 

I'd like to reconsider this approach in the case we need to scale the data filtering, we have to be careful of client side computations as per-frame performance it's very critical for us. In any case, take my LGTM.